pkg/docker/fast_push.go

```diff
@@ -26,6 +33,17 @@ const requirementsTarFile = "requirements.tar.zst"
 const schemeEnv = "R8_PUSH_SCHEME"
 const hostEnv = "R8_PUSH_HOST"
 
+type S3Config struct {
+	Key             string `json:"key"`
+	Bucket          string `json:"bucket"`
+	Endpoint        string `json:"endpoint"`
+	AccessKeyId     string `json:"access_key_id"`
+	SecretAccessKey string `json:"secret_access_key"`
+	SessionToken    string `json:"session_token"`
+	Expires         int64  `json:"expires"`
+	Uuid            string `json:"uuid"`
+}
+
 func FastPush(image string, projectDir string, command Command, ctx context.Context) error {
```

Comment:
This is a nit for later, but `ctx` really should be the first argument in the signature 👍🏼

Reply:
Resolved in [9f53230](https://github.com/replicate/cog/pull/2128/commits/9f53230ed8d7eac9d1102b7e605ec6e587b8e9f1)